packages/cli/src/modules/data-table/csv-parser.service.ts

```diff
@@ -109,10 +108,7 @@ export class CsvParserService {
 			const parser = parse({
 				columns: hasHeaders ? true : false,
 				skip_empty_lines: true,
-			});
-
-			createReadStream(filePath)
-				.pipe(parser)
+			})
 				.on('data', (row: Record<string, string> | string[]) => {
 					if (!hasHeaders && Array.isArray(row)) {
 						const processed = this.processRowWithoutHeaders(row, columnNames);
```

Comment:
This is outside the scope of this PR, but even though this handling is now streamed it still constructs the whole rows array in memory and only then inserts it into the DB, meaning that if the CSV is large we consume a lot of memory, which could be problematic when importing large data tables.

This should/could be refactored to only read a chunk of the file, insert it, drop it from memory and then move on to the next chunk, solving memory issues with _large_ data table imports.